src/browser/components/MainPage.jsx

```diff
@@ -526,7 +526,7 @@ export default class MainPage extends React.Component {
 
   handleTargetURLChange = (targetURL) => {
     clearTimeout(this.targetURLDisappearTimeout);
-    if (targetURL === '') {
+    if (targetURL === '' || this.parseDeeplinkURL(targetURL)) { // Do not show URL for internal links
```

Comment:
I think we need to also get the server we are currently in to pass as a second argument, that way we prevent hiding an internal link to another server

Reply:
@Willyfrog I think I've got it! See latest commit